apis/v1alpha1/backendpolicy_types.go

```diff
@@ -102,26 +121,28 @@ type BackendTLSConfig struct {
 	// Support: Extended
 	//
 	// +optional
-	CertificateAuthorityRef *LocalObjectReference `json:"certificateAuthorityRef,omitempty"`
+	CertificateAuthorityRef LocalObjectReference `json:"certificateAuthorityRef,omitempty"`
```

Comment:
this is optional, so should still be a pointer field?

Reply:
xref https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L50-L52 for other native k8s types with optional struct fields.